### DISCUSSION TITLE
Settings Sync between VS Code and Codespaces

### DISCUSSION THREAD

**Discussion by wei** [community/community#26036] [Codespaces] [answered]
<p>Is there anyway or any reason not to sync settings across vscode and codespaces? I logged into both via GitHub and it seems settings are shared between codespaces instances but not between vscode and codespaces or vice-versa.</p>
<p>Thank you!</p>

**Comment by Chuxel** [comment]
<p>Yes - Codespaces currently uses the VS Code Insiders channel. As a result, its settings sync with local VS Code Insiders rather than VS Code stable. Eventually we will move to VS Code stable and this behavior will go away.</p>
<p>If you don’t already have insiders installed, you can get it here: <a href="https://code.visualstudio.com/insiders/" rel="nofollow noopener">https://code.visualstudio.com/insiders/</a></p>
<p>It can live side-by-side with stable.</p>

**Comment by spaltrowitz** [comment, marked as answer]
Hi @wei, thanks for sharing this feedback. A feature related to this feedback has now been shipped! 🚢 https://github.blog/changelog/2023-05-04-codespaces-settings-sync-updates/ cc @Chuxel
